src/test/java/build/buf/protovalidate/ValidatorDifferentJavaPackagesTest.java

```diff
@@ -0,0 +1,171 @@
+// Copyright 2023 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package build.buf.protovalidate;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import build.buf.protovalidate.exceptions.ValidationException;
+import build.buf.validate.Violation;
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.google.protobuf.Message;
+import java.util.Collections;
+import java.util.List;
+import org.junit.Test;
+
+/**
```

Comment:
This provides full details on what we're fixing and tests both types of generated code and verifies it gets the same results.